src/utils/redis.js

```diff
@@ -0,0 +1,63 @@
+const { promisify } = require('util');
+const redis = require('redis');
+const jwt = require('jsonwebtoken');
+const { GeneralError } = require('@feathersjs/errors');
+
+let redisClient = false;
+
+function initializeRedisClient(app) {
+	const redisUrl = app.Config.data.REDIS_URI;
```

Comment:
Wollen wir hier die app reingeben, oder nur die REDIS_URI? Würde mir in Hinsicht auf den Test besser gefallen. Falls sich das bei comment ändert.

Reply:
wenn wir nur die REDIS_URI reingeben können wir nicht mehr app.configure nutzen, und fallen aus dem standart-schema von feathers raus. Ich würde mich hier an die Convention halten.

Reply:
man könnte natürlich überlegen die redis helper in irgendeiner Form an die app zu hängen...